### PR TITLE
[14.0][MIG] crm_timesheet

### PR DESCRIPTION
**Pull request by fshah-initos**
Depends on module project_timesheet_time_control https://github.com/OCA/project/pull/846

### CODE REVIEW

**Comment by fshah-initos** [comment]
Travis fails as the depended module PR is not yet merged.

**Comment by fshah-initos** [comment]
@rvalyi  can you please review the PR?

**Comment by fshah-initos** [comment]
It would be great if somone could add reviews here.

**Comment by fshah-initos** [comment]
It would be great if somone could add reviews here.

**Review by dsolanki-initos — APPROVED** [review]
Functional and code review done. LGTM

**Comment by fshah-initos** [comment]
@Abranes can you please add the reviews here.

**Comment by fshah-initos** [comment]
It would be great if we could get the reviews here

**Comment by fshah-initos** [comment]
It would be great if we could get the reviews here

**Comment by fshah-initos** [comment]
It would be great if we could get the reviews here

**Comment by rvalyi** [comment]
This PR is all the rage but meanwhile its dependency is still no passing the tests and not reviewed https://github.com/OCA/project/pull/772

**Comment by hkapatel-initos** [comment]
It would be great if we could get the reviews here

**Comment by hkapatel-initos** [comment]
> Depends on module project_timesheet_time_control #[http://github.com/OCA/project/pull/772](https://github.com/OCA/project/pull/772)

Dependensies 
https://github.com/OCA/project/pull/846

**Comment by hkapatel-initos** [comment]
runbot is not happy .
@rvalyi it would be nice if you check this module.

**Comment by hkapatel-initos** [comment]
@rvalyi it would be nice if you review this pull.

**Comment by hkapatel-initos** [comment]
it would be great if someone add reviews here.

**Comment by fshah-initos** [comment]
It would be nice if get reviews here

**Review by victoralmau — COMMENTED** [review]
Code review OK.
Please rebase to check Travis.

**Review by victoralmau — APPROVED** [review]

File: crm_timesheet/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_timesheet_switch,access_hr_timesheet_switch,model_hr_timesheet_switch,base.group_user,1,1,1,1
```
Comment (pedrobaeza) [inline crm_timesheet/security/ir.model.access.csv]: This should be timesheet user, not any user.

**Comment by pedrobaeza** [comment]
It seems you have removed your name from contributors and added an extra empty line in the test file.

**Comment by hkapatel-initos** [comment]
> It seems you have remove your name from contributors and added an extra empty line in the test file.

updated.

File: crm_timesheet/tests/test_account_analytic_line.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+from odoo import exceptions
+from odoo.tests.common import SavepointCase
+
+
+class AccountAnalyticLineCase(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        admin = cls.env.ref("base.user_admin")
+
```
Comment (pedrobaeza) [inline crm_timesheet/tests/test_account_analytic_line.py]: Remove this extra empty line that doesn't add anything

Comment (hkapatel-initos) [reply crm_timesheet/tests/test_account_analytic_line.py]: Done

File: crm_timesheet/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
+* `Tecnativa <https://www.tecnativa.com>`_
+  * Antonio Espinosa
```
Comment (pedrobaeza) [inline crm_timesheet/readme/CONTRIBUTORS.rst:3]: Sorry, this is not your fault, but I'm seeing this requires an extra line in between this line and the above for proper RST syntax.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**